python/src/main/java/org/navgurukul/playground/editor/PythonEditorFragment.kt

```diff
@@ -272,6 +276,29 @@ class PythonEditorFragment : BaseFragment() {
 
     }
 
+    private fun showFileNotSavedDialog(){
+        val fileNotSavedDialog = AlertDialog.Builder(requireContext())
+        fileNotSavedDialog.setTitle("File not saved")
```

Comment:
fileNotSavedDialog.apply {
 setTitle()
 setMessage()
 .
 .
}
You may also use this for a better readability, also looks a bit cleaner this way and saves the redundancy of variable name everytime.